Apps/WebClient/src/ClientApp/src/models/laboratory.ts

```diff
@@ -32,6 +32,38 @@ export interface Covid19LaboratoryTest {
     loincName: string | null;
 }
 
+// laboratory order model
+export interface LaboratoryOrder {
+    id: string;
+    phn: string | null;
+    orderingProviderIds: string | null;
+    orderingProviders: string | null;
+    reportingLab: string | null;
+    location: string | null;
+    ormOrOru: string | null;
+    messageDateTime: StringISODateTime;
+    messageId: string | null;
+    additionalData: string | null;
+    reportAvailable: boolean;
+    labResults: Covid19LaboratoryTest[];
+}
+
+// laboratory test model
+export interface LaboratoryTest {
+    id: string;
+    testType: string | null;
+    outOfRange: boolean;
+    collectedDateTime: StringISODateTime;
+    testStatus: string | null;
+    resultDescription: string[];
+    resultLink: string | null;
+    labResultOutcome: string | null;
+    receivedDateTime: StringISODateTime;
+    resultDateTime: StringISODateTime;
+    loinc: string | null;
+    loincName: string | null;
+}
```

Comment:
These models should be updated to match the ones in the **Laboratory** project under `src/Models/LaboratoryOrder.cs` and `src/Models/LaboratoryTest.cs`.

Reply:
And `labResults` (now `laboratoryTests`) on `LaboratoryOrder` should be of type `LaboratoryTest`.